Python/Setup/swix/Packages/Microsoft.PythonTools.Profiling.Vsix.swixproj

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Just to make it clear (to me at least) that this package is the vsix file. e.g. `Microsoft.PythonTools.Profiling` is a "component" (that can be optionally enabled/disabled by users; `<OutputType>manifest</OutputType>` and `vs.package.type component`), while `Microsoft.PythonTools.Profiling.Vsix` is a "VSIX package" (`<OutputType>vsix</OutputType>`). Other packages tend to have `.Exe` or `Msi` at the end when they solely exist for that payload.

(The OutputType and SwixProjectType properties are set in the SetupProjectBefore/After.settings files, since I arranged the projects to be in separate dirs.)